db/sql/045_create_vat_rates.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS vat_rates (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR(32),
+	rate NUMERIC,
+
+	CHECK (length(trim(name)) > 0),
+	CHECK (rate >= 0)
+);
+CREATE INDEX IF NOT EXISTS idx_vat_rates_id ON vat_rates (id);
```

Comment:
`PRIMARY KEY` will create an index so I think this extra index is superfluous

Reply:
fixed